node_app/client/index.html

```diff
@@ -98,4 +189,4 @@ <h5>Upload your track</h5>
 
```

Comment:
add Bootstrap JS here 

```suggestion
    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js"
            integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p"
            crossorigin="anonymous"></script>
```

Reply:
hei, I still have to make it Responsive, so far it is just a  workable structure.

I'm gonna add responsiveness at last, when we agree on the style